spec/factories/recipe.rb

```diff
@@ -0,0 +1,58 @@
+FactoryBot.define do
+  factory :recipe do
+
+    trait :scrambled_eggs do
+      name 'scrambled eggs'
+      instructions 'Scramble eggs'
+
+      after_build do |recipe|
+        recipe.ingridients << FactoryBot.create(:ingridient, :egg)
```

Comment:
Don't we need to pass specific amount?

```ruby
recipe.ingridients << FactoryBot.create(:ingridient, :egg, unit_amount: 200)
```

This should be applied to the whole file.